comments_blackbox_test.go

```diff
@@ -240,3 +240,27 @@ func (s *CommentsSuite) TestUpdateCommentWithOtherUser() {
 	userSvc, _, _, commentsCtrl := s.securedControllers(testsupport.TestIdentity2)
 	test.UpdateCommentsForbidden(s.T(), userSvc.Context, userSvc, commentsCtrl, commentId, updateCommentPayload)
 }
+
+func (s *CommentsSuite) TestDeleteCommentWithSameAuthenticatedUser() {
+	// given
+	workitemId := s.createWorkItem(testsupport.TestIdentity)
+	commentId := s.createWorkItemComment(testsupport.TestIdentity, workitemId)
+	userSvc, _, _, commentsCtrl := s.securedControllers(testsupport.TestIdentity)
+	test.DeleteCommentsOK(s.T(), userSvc.Context, userSvc, commentsCtrl, commentId)
+}
+
+func (s *CommentsSuite) TestDeleteCommentWithoutAuth() {
+	// given
+	workitemId := s.createWorkItem(testsupport.TestIdentity)
+	commentId := s.createWorkItemComment(testsupport.TestIdentity, workitemId)
+	userSvc, commentsCtrl := s.unsecuredController()
+	test.DeleteCommentsUnauthorized(s.T(), userSvc.Context, userSvc, commentsCtrl, commentId)
+}
+
+func (s *CommentsSuite) TestDeleteCommentWithOtherAuthenticatedUser() {
+	// given
+	workitemId := s.createWorkItem(testsupport.TestIdentity)
+	commentId := s.createWorkItemComment(testsupport.TestIdentity, workitemId)
+	userSvc, _, _, commentsCtrl := s.securedControllers(testsupport.TestIdentity2)
+	test.DeleteCommentsUnauthorized(s.T(), userSvc.Context, userSvc, commentsCtrl, commentId)
```

Comment:
@Ritsyy no, "BadRequest" should be used when a request parameter is invalid (eg: expecting a number and received a string) or missing. "Forbidden" is what should be used. 
See https://github.com/almighty/almighty-core/blob/master/comments_blackbox_test.go#L241,  
https://github.com/almighty/almighty-core/blob/master/design/comments.go#L142 and